autoarray/util/grid_util.py

```diff
@@ -762,3 +762,25 @@ def grid_upscaled_1d_from(
                 upscale_index += 1
 
     return grid_upscaled_1d
+
+def mask_of_points_within_radius(radius: float, centre: (float, float), grid: np.ndarray):

```

Comment:
The type of a tuple should actually be Tuple[float, float] where Tuple is imported from typing